extensions/amp-analytics/0.1/iframe-transport.js

```diff
@@ -37,31 +37,34 @@ export let FrameData;
  */
 export class IframeTransport {
   /**
-   * @param {!Window} win
+   * @param {!Window} ampWin The window object of the AMP document
    * @param {!string} type The value of the amp-analytics tag's type attribute
    * @param {!JsonObject} config
+   * @param {!string} ampAdResourceId The resourceID of the enclosing amp-ad tag
    */
-  constructor(win, type, config) {
-    /** @private @const {!Window} win */
-    this.win_ = win;
+  constructor(ampWin, type, config, ampAdResourceId) {
```

Comment:
replacing id with `ampAdResourceId` limit the use of iframe transport to ad related analytics only. Is this what we want?

Reply:
Hmm...good point. Will rename.

Reply:
Done